insights/parsers/lpstat.py

```diff
@@ -42,10 +24,31 @@
 
 @parser(Specs.lpstat_p)
 class LpstatPrinters(CommandParser):
-    """Class to parse ``lpstat -p`` command output.
+    """
+    Class to parse ``lpstat -p`` command output. Parses the output of ``lpstat -p``, to get locally configured printers.
+    Current available printer states are:
+
+    * IDLE (``PRINTER_STATUS_IDLE``)
+    * PROCESSING (``PRINTER_STATUS_PROCESSING``) -- printing
+    * DISABLED (``PRINTER_STATUS_DISABLED``)
+    * UNKNOWN (``PRINTER_STATUS_UNKNOWN``)
 
-    Raises:
+    Sample output of the command::
+
+        printer idle_printer is idle.  enabled since Fri 20 Jan 2017 09:55:50 PM CET
+        printer disabled_printer disabled since Wed 15 Feb 2017 12:01:11 PM EST -
+        reason unknown
+
+    Raises::
         ValueError: Raised if any error occurs parsing the content.
+
+    Examples::
```

Comment:
Better use a single colon here for `Examples`, and `Raises`

Reply:
Updated